dashboard/app/controllers/home_controller.rb

```diff
@@ -167,9 +167,11 @@ def terms_and_privacy
 
       unless current_user.donor_teacher_banner_dismissed
         donor_banner_name = current_user.school_donor_name
```

Comment:
Is it possible to get rid of `donor_banner_name` here and on 173?

Reply:
Thanks for the push looks like we can remove it.